be/src/storage/rowset/array_column_iterator.h

```diff
@@ -56,6 +55,9 @@ class ArrayColumnIterator final : public ColumnIterator {
     std::unique_ptr<ColumnIterator> _null_iterator;
     std::unique_ptr<ColumnIterator> _array_size_iterator;
     std::unique_ptr<ColumnIterator> _element_iterator;
+    ColumnAccessPath* _path;
```

Comment:
what's the path meaning for array column? like these a[1], a[a[4]]?

Reply:
yes, like a[1], the path will be `a/INDEX`, I think i can add special paramters in path, to describe the `[]`'s value, but I have not implement now